Server/myServer.py

```diff
@@ -1,9 +1,13 @@
-from flask import Flask
+from flask import Flask, send_from_directory
 
-app = Flask(__name__)
+app = Flask(__name__, static_url_path='/', static_folder='.\\..\\react-paw-mailmerge\\build')
```

Comment:
If we use `os.path` to construct the path to the static folder then it'll work on both Windows and Linux
https://docs.python.org/3/library/os.path.html#module-os.path
```python
import os
static_folder_path = os.path.join('..', 'react-paw-mailmerge', 'build')
```

Reply:
Nice catch! Thanks for pointing this out.